packages/cli/fragments/fragment-svelte-ts/vite.config.ts

```diff
@@ -1,9 +1,10 @@
 import { defineConfig } from "vite";
 import { svelte } from "@sveltejs/vite-plugin-svelte";
+import sveltePreprocess from "svelte-preprocess";
 
 // https://vitejs.dev/config/
 export default defineConfig({
-  plugins: [svelte()],
+  plugins: [svelte({ preprocess: [sveltePreprocess({ typescript: true })] })],
```

Comment:
run `pnpm format` on the repo so this line get formatted correctly

Reply:
Just ran it a second ago, doesn't seem to have changed anything. Ran `pnpm format:check` and I get the following:

```bash
> @ format:check /Users/me/Desktop/create-tauri-app
> prettier --check --end-of-line=auto "./**/*.{js,jsx,ts,tsx,html,css,json,vue}"

Checking formatting...
All matched files use Prettier code style!
```

Is it possible there is something wrong with my Prettier installation? I attempted to reinstall it but the result remains the same. I got this same result when I ran `pnpm format` before the commit—I was surprised by it. Is it possible this qualifies as conforming to Prettier code style?

Reply:
yeah, seems like the line doesn't exceed the 80-character width so it won't wrap it. 

Than hand-formatted version sounds good, thanks.